### PR TITLE
Missing French in service settings and 'Receive text message' page

### PR DESCRIPTION
**Pull request by anikbrazeau**
- Added FR content for 'Receive text message' page in settings
- Translated service settings

### CODE REVIEW

**Comment by anikbrazeau** [comment]
[Missing French in Settings: "Receive text message" page, change links, and other variables](https://trello.com/c/QoDqC4Up/41-missing-french-in-settings-receive-text-message-page-change-links-and-other-variables)

**Comment by AntoineAugusti** [comment]
Service settings in 🇫🇷 
Screen Shot 2020-09-25 at 15 12 43

File: app/templates/views/service-settings.html [hunk 1]
```diff
@@ -351,9 +351,9 @@ <h2 class="heading-medium">{{ _('Platform admin settings') }}</h2>
           {% for permission in service_permissions %}
             {% if not service_permissions[permission].requires or current_service.has_permission(service_permissions[permission].requires) %}
               {% call row() %}
-                {{ text_field(service_permissions[permission].title)}}
+                {{ text_field(_(service_permissions[permission].title))}}
                 {{ boolean_field(current_service.has_permission(permission)) }}
-                {{ edit_field('Change', url_for(service_permissions[permission].endpoint or '.service_set_permission', service_id=current_service.id, permission=permission if not service_permissions[permission].endpoint else None)) }}
+                {{ edit_field(change_txt, url_for(service_permissions[permission].endpoint or '.service_set_permission', service_id=current_service.id, permission=permission if not service_permissions[permission].endpoint else None)) }}
```
Comment (brdunfield) [inline app/templates/views/service-settings.html:356]: 👍 Great catch

File: app/translations/csv/en.csv [hunk 2]
```diff
@@ -1168,3 +1168,17 @@
...
+"Your service can receive text messages sent to {}.",""
+"You can set up callbacks for received text messages on the <a href='{}'>API integration page</a>.",""
+"We’ll create a unique phone number for your service that they can reply to.",""
+"Text messages you send will come from this phone number instead of your sender name (currently {}).",""
+"You can still send text messages from a sender name if you need to, but users will not be able to reply to those messages.",""
+"You can see and reply to the messages you receive when you sign in to GC Notify, or get them using the API.",""
+"<a href='{}'>Contact us</a> if you want to be able to receive text messages from your users.",""
+"<a href='{}'>Contact us</a> if you want to switch this feature off.",""
+"It does not cost you anything to receive text messages. Users will pay their standard text message rate.",""
+"Government of Canada",
```
Comment (brdunfield) [inline app/translations/csv/en.csv:1180]: missing the ,"" at the end on this line

**Review by brdunfield — APPROVED** [review]
👍 LGTM! 🦑

**Pull request merged**